Makefile

```diff
@@ -109,14 +112,23 @@ lint: lint-dependencies lint-all
 ############################################################
 # test section
 ############################################################
+GOSEC = $(shell pwd)/bin/gosec
 
 test:
-	@go test ${TESTARGS} `go list ./... | grep -v test/e2e`
+	go test $(TESTARGS) `go list ./... | grep -v test/e2e`
+
+test-coverage: TESTARGS = -json -coverprofile=coverage_unit.out
+test-coverage: test
 
 test-dependencies:
 	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v$(KBVERSION)/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
 	sudo mv /tmp/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH) /usr/local/kubebuilder
 
+gosec-install:
+	curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(shell pwd)/bin
```

Comment:
Do you think you could use `go get` to compile it or at the very least target a specific tag/commit? It seems a little dangerous to be running a script right from the master branch in the event the repo gets compromised.

Reply:
Yes, I can do that!

Reply:
Although, this is consistent with what CICD uses in Prow, so if the repo is compromised we're all in trouble...

Reply:
@dhaiducek but we can always strive to do better.  :)

Reply:
Indeed we can 🙂 Updated!

I tried the `go get` method, but it doesn't spit out a version when running `gosec --version` that way, so I think the tarball may be the way to go.